config/initializers/dragonfly.rb

```diff
@@ -0,0 +1,58 @@
+require 'dragonfly'
+require 'dragonfly/s3_data_store'
+
+include ActionView::Helpers::NumberHelper
+
+# Configure
+Dragonfly.app.configure do
+  plugin :imagemagick
+
+  secret Rails.application.secrets.dragonfly_secret
+
+  secret "4535d2f3eae9919e9d3c642ba751205abe26f096367cd4611daadb50af749fc2"
```

Comment:
This should definitely go away in favor of line 10 -- going through Rails application secrets.